Server/src/services/getDatasetService.ts

```diff
@@ -211,10 +216,43 @@ export class retrieveData {
      * @param userReceived
      * Account ID: number
      */
-    async getUserSavedDatasets(userReceived: number) {
+    async getUserSavedDatasets(userReceived: string) {

```

Comment:
by the way, if we arent sure if user received id is a string or a number we can write the type as string | number <- this accepts both

Reply:
Will be number as userID can be used now